lib/Locale/Country/Extra.pm

```diff
@@ -2,7 +2,7 @@ package Locale::Country::Extra;
 use strict;
 use warnings;
 
-our $VERSION = '1.0.1';
+our $VERSION = '1.0.2';
```

Comment:
I remember `dzil release` will increase the version  number automatically, we needn't increase it by hand.